dev-python/typer/typer-0.15.3.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..13} )
+DISTUTILS_USE_PEP517=pdm-backend
+
+inherit distutils-r1 shell-completion pypi
+
+DESCRIPTION="Build great CLIs. Easy to code. Based on Python type hints."
+HOMEPAGE="
+	https://typer.tiangolo.com/
+	https://github.com/tiangolo/typer
+	https://pypi.org/project/typer/
+"
+SRC_URI="$(pypi_sdist_url "${PN}" "${PV}")"
```

Comment:
Why?

Reply:
I push other fixes for SRC_URIs too (in pyspellchecker and safety-schemas.